prow/cmd/tackle/main.go

```diff
@@ -626,15 +628,27 @@ func clientConfig(context string) (*rest.Config, error) {
 }
 
 func ingress(kc *kubernetes.Clientset, ns, service string) (url.URL, error) {
+	var obj interface{}
```

Comment:
aka

```golang
var ing networking.IngressList
if networkingIngressAvail(kc) {
  ing = kc.NetworkingV1beta1().Ingresses(ns).List()
} else {
  ing = toIngress(kc.ExtensionsV1beta1().Ingresses(ns),.List())
}
```

Reply:
@fejta - updated the `toIngress(...)` function to explicitly accept an *extensions* IngressList and return a *networking* IngressList removing the type assertion.